Functions/InputOutput/xASL_io_dcm2nii.m

```diff
@@ -150,6 +150,36 @@
     %   -x : crop (y/n, default n)
     %   -z : gz compress images (y/i/n/3, default n) [y=pigz, i=internal:miniz, n=no, 3=no,3D]
 
+	% -1..-9 : gz compression level (1=fastest..9=smallest, default 6)

```

Comment:
Why do we add this list, also: do these parameters change between dcmnii versions?

Reply:
Yes - there was no need to have any of those helps. I removed them all. See 43d03f7b